src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c

```diff
@@ -105,83 +104,65 @@ const ActorInit En_Bigpo_InitVars = {
     { COLTYPE_HIT3, AT_NONE | AT_TYPE_ENEMY, AC_NONE | AC_TYPE_PLAYER, OC1_NONE | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_CYLINDER, },
     { ELEMTYPE_UNK0, { 0xF7CFFFFF, 0x00, 0x10 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_ON | TOUCH_SFX_NORMAL, BUMP_ON | BUMP_HOOKABLE, OCELEM_ON, },
     { 35, 100, 10, { 0, 0, 0 } },
-}; */
+};
 
-extern CollisionCheckInfoInit D_80B6503C;
-//static CollisionCheckInfoInit D_80B6503C = { 0x0A000023, 0x00643200};
-//static u32 D_80B6503C[] = { 0x0A000023, 0x00643200};
+static CollisionCheckInfoInit D_80B6503C = { 10, 35, 100, 50};
 
-extern DamageTable D_80B65044;
-/* static DamageTable D_80B65044 = {
+static DamageTable D_80B65044 = {
     0x00, 0x01, 0x00, 0x01, 0x01, 0x01, 0x00, 0xF0, 0x01, 0x01, 0x00, 0x01, 0x01, 0x42, 0x01, 0x01, 
     0x01, 0x02, 0x00, 0x00, 0x00, 0x00, 0x01, 0x01, 0x01, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 
-}; */
+}; 
 
-static InitChainEntry D_80B65064;
-/* static InitChainEntry D_80B65064[] = { //sInitChain
+static InitChainEntry D_80B65064[] = { //sInitChain
     ICHAIN_S8(hintId, 90, ICHAIN_CONTINUE),
     ICHAIN_F32(targetArrowOffset, 3200, ICHAIN_STOP),
-}; */
+}; 
 
-extern Vec3f D_80B6506C;
-//static Vec3f D_80B6506C = { 0.0f, 3.0f, 0.0f};
+static Vec3f D_80B6506C = { 0.0f, 3.0f, 0.0f};
 
-extern u8 D_80B65078[];
-/* static u8 D_80B65078[] = {
+static u8 D_80B65078[] = {
     0xFF, 0x04, 0xFF, 0x00, 
     0xFF, 0x01, 0xFF, 0x02, 
     0x05, 0x03, 0x00, 0x00, 
-}; */
+}; 
 
-extern Vec3f D_80B65084;
-/* static Vec3f D_80B65084[] = {
+static Vec3f D_80B65084[] = {
     { 2000.0f, 4000.0f, 0.0f,},
-    {-1000.0f, 1500.0f, -1000.0f,},
+    {-1000.0f, 1500.0f, -2000.0f,},
     {-1000.0f, 1500.0f, 2000.0f,},
-}; */
+}; 
 
-#ifdef NON_MATCHING
-// non-matching: some instructions out of order but looks like it should be equiv
-void EnBigpo_Init(Actor* thisx, GlobalContext *globalCtx) {
+void EnBigpo_Init(Actor* thisx, GlobalContext *globalCtx2) {
+    GlobalContext* globalCtx = globalCtx2;
     EnBigpo* this = (EnBigpo*) thisx;
     EnBigpoFireParticle* fires;
     s32 i;
     u16 parms;
 
     Actor_ProcessInitChain(&this->actor, &D_80B65064);
 
-    // issue:  the params look-up wants to save the params to two t registers, one modified, one not
-    // then it saves the param, and immediately re-loads it wtf
-    // because its duping the globalcontext load instead of the params load, I can only assume its filling a void
-    this->switchFlags = this->actor.params;
-    //this->switchFlags = this->switchFlags >> 8;
-    this->switchFlags = (u8)(this->switchFlags);
-    this->actor.params &= 0xFF;
-    //if (this->actor.params == 9){} // almost like it needs a a different params for a different scope
-    //this->actor.params = (u8)this->actor.params;
-    if (this->actor.params == ENBIGPO_POSSIBLEFIRE) {
-    //if (parms == ENBIGPO_POSSIBLEFIRE) {
+    this->switchFlags = (u8)(thisx->params >> 8);
+    thisx->params &= 0xFF;
```

Comment:
Ohhhhhh... thisx.

I've heard its been required to match a few times, haven't gotten it myself before now though, interesting.